forc/src/cli/commands/test.rs

```diff
@@ -44,56 +45,73 @@ pub(crate) fn exec(cmd: Command) -> Result<()> {
 
     // Eventually we'll print this in a fancy manner, but this will do for testing.
     match tested {
-        forc_test::Tested::Workspace => unimplemented!(),
-        forc_test::Tested::Package(pkg) => {
-            let succeeded = pkg.tests.iter().filter(|t| t.passed()).count();
-            let failed = pkg.tests.len() - succeeded;
-            let mut failed_test_details = Vec::new();
-            for test in &pkg.tests {
-                let test_passed = test.passed();
-                let (state, color) = match test_passed {
-                    true => ("ok", Colour::Green),
-                    false => ("FAILED", Colour::Red),
-                };
-                info!(
-                    "      test {} ... {} ({:?})",
-                    test.name,
-                    color.paint(state),
-                    test.duration
-                );
-
-                // If the test is failing, save details.
-                if !test_passed {
-                    let details = test.details()?;
-                    failed_test_details.push((test.name.clone(), details));
-                }
-            }
-            let (state, color) = match succeeded == pkg.tests.len() {
-                true => ("OK", Colour::Green),
-                false => ("FAILED", Colour::Red),
-            };
-            if failed != 0 {
-                info!("\n   failures:");
-                for (failed_test_name, failed_test_detail) in failed_test_details {
-                    let path = &*failed_test_detail.file_path;
-                    let line_number = failed_test_detail.line_number;
-                    info!(
-                        "      - test {}, {:?}:{} ",
-                        failed_test_name, path, line_number
-                    );
-                }
-                info!("\n");
+        forc_test::Tested::Workspace(pkgs) => {
+            for pkg in pkgs {
+                let built = &pkg.built.pkg_name;
+                info!("\n   testing -- {built}\n");
```

Comment:
"testing" might imply that the testing begins after this output, but testing has already completed by the time we reach this point. Perhaps this should be:

```suggestion
                info!("\n   tested -- {built}\n");
```